mirage/views/file.ts

```diff
@@ -46,9 +47,13 @@ export function uploadToRoot(this: HandlerContext, schema: Schema) {
     const uploadAttrs = this.request.requestBody;
     const { parentID, fileProviderId } = this.request.params;
     const { name, kind } = this.request.queryParams;
-    let node;
+    let isPreprint = false;
+    let node: any | PreprintModel;
     if (this.request.url.includes('draft_nodes')) {
         node = schema.draftNodes.find(parentID);
+    } else if (this.request.url.includes('file')) {
```

Comment:
Is this right? Or should it be `.includes('preprint')`?

Reply:
Right, but there are lots of reasons why upload to root might have a 'file' in the url, but presumably the only time you need it to set it to be a 'preprint' is if preprint is in the url, right?